play-json/jvm/src/main/scala/play/api/libs/json/EnvWrites.scala

```diff
@@ -207,6 +209,28 @@ trait EnvWrites {
       JsNumber(BigDecimal valueOf t.toEpochMilli)
   }
 
+  /** Serializer of Java Duration as a number of milliseconds. */
+  val javaDurationMillisWrites: Writes[JDuration] =
+    Writes[JDuration] { d => JsNumber(d.toMillis) }
+
+  /**
+   * Serializer of Java Duration using ISO representation
+   * (e.g. PT1S for 1 second).
+   */
+  implicit val javaDurationWrites: Writes[JDuration] =
+    Writes[JDuration] { d => JsString(d.toString) }
+
+  /** Serializer of Java Period as a number of days. */
+  val javaPeriodDaysWrites: Writes[Period] =
+    Writes[Period] { d => JsNumber(d.getDays) }
```

Comment:
`getDays` returns only the `days` field in the `Period`. I'd think you'd want to convert the months and years to days and include those at well. You could use `Duration.from(period).toDays`.

It would also be useful to have constants for years and months like I suggested for the `Reads`